docs/overview.md

```diff
@@ -1,101 +1,98 @@
-
-When the Parachain validator calls the `validate_block` function, it passes the PoVBlock (See [Block
-building](#block-building) for more information) and the parent header of the Parachain that is
-stored on the relay chain. From the PoVBlock witness data, Cumulus reconstructs the partial trie.
-This partial trie is used as storage while executing the block. Cumulus also redirects all storage
-related host functions to use the witness data storage. After the setup is done, Cumulus calls
-`execute_block` with the transactions and the header stored in the PoVBlock. On success, the new
-Parachain header is returned as part of the `validate_block` result.
-
-## Node
-
-Parachains support light-clients, full nodes, and authority nodes. Authority nodes are called
-Collators in the Polkadot ecosystem. Cumulus provides the consensus implementation for a
-Parachain and the block production logic.
-
-The Parachain consensus will follow the relay chain to get notified about which Parachain blocks are
-included in the relay-chain and which are finalized. Each block that is built by a Collator is sent
-to a validator that is assigned to the particular Parachain. Cumulus provides the block production
-logic that notifies each Collator of the Parachain to build a Parachain block. The
-notification is triggered on a relay-chain block import by the Collator. This means that every
-Collator of the Parachain can send a block to the Parachain validators. For more sophisticated
-authoring logic, the Parachain will be able to use Aura, BABE, etc. (Not supported at the moment)
-
-A Parachain Collator will join the Parachain network and the relay-chain network. The Parachain
-network will be used to gossip Parachain blocks and to gossip transactions. Collators will only
-gossip blocks to the Parachain network that have a high chance of being included in the relay
-chain. To prove that a block is probably going to be included, the Collator will send along side
-the notification the so-called candidate message. This candidate message is issued by a Parachain
-validator after approving a block. This proof of possible inclusion prevents spamming other collators
-of the network with useless blocks.
-The Collator joins the relay-chain network for two reasons. First, the Collator uses it to send the
-Parachain blocks to the Parachain validators. Secondly, the Collator participates as light/full-node
-of the relay chain to be informed of new relay-chain blocks. This information will be used for the
-consensus and the block production logic.
+When compiled into a WASM binary and registered with a set of Validator Nodes, the substrate node can join as a Parachain. It does so primarily through a `validate_block` macro call implemented as a Cumulus pallet, which takes the `Block` and `Executive` type. `Executive` is a declared type that dispatches incoming calls to the correct module at runtime; it is an orchestration component—more on `Block` below.
+
+## Consensus
```

Comment:
The remainder of this doc, while a _great_ initiative, I think I see a few factual errors. I would propose be removed completely in favor of the official documentation that is formally maintained:

- https://wiki.polkadot.network/docs/en/learn-parachains
- https://wiki.polkadot.network/docs/en/learn-parachains#resources (for blogs with deeper tech details)

What from here should be migrated into those?

Reply:
I wrote this as part of a job application for Parity. It never went through a technical review process.
If you want to point out errors, I can make the changes necessary.

As for where it gets hosted, totally your call.